cypress/integration/e2eTests/homePage.js

```diff
@@ -0,0 +1,26 @@
+class HomePage {
+  visit() {
+    cy.visit(Cypress.config().baseUrl);
+  }
+
+  getFirstLineText() {
+    return cy.get('h1 ~ p:nth-child(2)');
+  }
+
+  getSecondLineText() {
+    return cy.get('h1 ~ p:nth-child(3)');
+  }
+
+  getLogo() {
+    return cy.get('div.am-navbar-left > a > img')
+  }
+  getNeedHelpElement() {
+    return cy.get('div:nth-child(1) > a > div');
```

Comment:
What's the strategy to find the right DOM elements for tests? 
Some of these look very brittle, depending on the exact HTML structure, e.g. "first child div of link within div"etc... 
Since we use Styled Components we don't have specific class names for the components either (unless we enable this through a Babel Macro / plugin which should be possible) 
Could it be that Cypress docs recommend to use `data-` attributes for this? I remember reading that in testing documentation but not sure if it was Cypress.

Reply:
Hi @robinv85,
Yes you're right. It's recommended to use data- attributes to have test more resilient. Is it possible to enable/add data-* attributes?

Reply:
Yeah I'm not a big fan of adding these data attributes everywhere but I guess if QA engineers want to do it it's ok, if we could use the StyledComponents plugin to automatically generate the class-names based on Component / File name I would also like that, maybe even more, then we could used more classes.